db/schema.rb

```diff
@@ -256,6 +256,19 @@
     t.boolean  "locked",                  default: false
   end
 
+  create_table "fosters", force: :cascade do |t|
```

Comment:
Where does this Foster table keep coming from?

Reply:
I am not sure but anytime I see that the migrations don't work or whatever is going on is really weird, I clone a clean copy to make these changes. I have a feeling this is how I setup or seed the database? I am not entirely sure though.